tiledb/sm/c_api/tiledb.h

```diff
@@ -4521,6 +4521,100 @@ TILEDB_EXPORT int32_t tiledb_vfs_is_empty_bucket(
 TILEDB_EXPORT int32_t tiledb_vfs_is_bucket(
     tiledb_ctx_t* ctx, tiledb_vfs_t* vfs, const char* uri, int32_t* is_bucket);
 
+/**
+ * Creates an Azure storage container.
+ *
+ * **Example:**
+ *
+ * @code{.c}
+ * tiledb_vfs_create_azure_container(ctx, vfs, "azure://tiledb");
```

Comment:
In other words, use `create_bucket` `remove_bucket` etc. as long as the concepts map sufficiently.

Reply:
+1 on this. I am happy to discuss further in case we need any renaming (e.g., use something else instead of `bucket`, if that buys us name consistency across all backends).

Reply:
As discussed in the #capi channel, I've updated this patch to re-use the existing "bucket" interfaces (e.g. create_bucket, is_bucket) and have the VFS switch on either S3 or Azure for the implementation.